src/System.IO.Ports/tests/SerialPort/AbortOnError.cs

```diff
@@ -28,7 +28,7 @@ public class AbortOnError : PortsTest
         /// 3. Verify that the fAbortOnError flag is clear
         /// 
         /// </summary>
-        [ConditionalFact(nameof(HasOneSerialPort))]
+        [ConditionalFact(nameof(HasOneSerialPort), nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotUap))]
```

Comment:
You likely need an rd.xml. Look for other examples eg 
C:\git\corefx\src\System.Runtime\tests\Resources\System.Runtime.Tests.rd.xml